### PR TITLE
Mark react sdk beta

### PR DESCRIPTION
**Pull request by mooreds**
added links to blog posts as well.

### CODE REVIEW

File: site/_posts/2021-11-11-how-to-authenticate-your-react-app.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ Whenever you build a website that allows a user to create their own account, sec
 
 <!--more-->
 
+{% include _callout-tip.liquid
+content=
+ "This blog post walks through low-level details of OAuth in React. If you want to add login, logout, and registration buttons to your React application, using pre-built buttons, hooks, or higher-order components, you should take a look at our [React SDK](/docs/v1/tech/client-libraries/react-sdk)."
```
Comment (bhalsey) [inline site/_posts/2021-11-11-how-to-authenticate-your-react-app.md:19]: should we mark it as beta here?

File: site/_posts/2020-03-10-securely-implement-oauth-in-react.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ In this post, we'll walk step-by-step through implementing the OAuth Authorizati
 
 {% include _callout-tip.liquid
 content=
- "This blog post has been [superseded by a newer React/OAuth tutorial](/blog/2021/11/11/how-to-authenticate-your-react-app)."
+ "This blog post walks through low-level details of OAuth in React. If you want to add login, logout, and registration buttons to your React application, using pre-built buttons, hooks, or higher-order components, you should take a look at our [React SDK](/docs/v1/tech/client-libraries/react-sdk)."
```
Comment (bhalsey) [inline site/_posts/2020-03-10-securely-implement-oauth-in-react.md:19]: should we mark it as beta here?

**Review by bhalsey — APPROVED** [review]

File: site/_posts/2020-03-10-securely-implement-oauth-in-react.md (hunk 2 above)
Comment (mooreds) [reply site/_posts/2020-03-10-securely-implement-oauth-in-react.md]: yup

File: site/_posts/2021-11-11-how-to-authenticate-your-react-app.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2021-11-11-how-to-authenticate-your-react-app.md]: yup

**Pull request merged**